.github/CONTRIBUTING.md

```diff
@@ -69,8 +69,7 @@ Submitting a pull request
 With over 5000 natives, it's hard for us to manually review and test all contributions. We love to see a well tested example to validate your change in the pull request description. Doing so will allow us to review your change much faster, and avoids untested or wrong information. This includes:
 
 - Code to test and validate your change with
-- Screenshots showcasing what the change is about
-- Source material of used information
```

Comment:
Source material of used information is needed, people should provide credit where it's due.